webgpu.h

```diff
@@ -209,6 +215,14 @@ typedef enum WGPUPrimitiveTopology {
     WGPUPrimitiveTopology_Force32 = 0x7FFFFFFF
 } WGPUPrimitiveTopology;
 
+typedef enum WGPUSType {
+    WGPUSType_Invalid = 0x00000000,
```

Comment:
Made this a regular enum and added the "0" value is invalid so that people using the C API know they can't leave it undefined. (we could have good error messages on invalid sType that say you forgot to set it).